src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Color ballColor = colorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100);
```

Comment:
Don't create a new instance of `Random` every time the method is called. Instead, create a single static final instance as a class-level field and reuse it.